src/converters/jsonToProtobufFactory.js

```diff
@@ -0,0 +1,35 @@
+const camelCase = require('lodash.camelcase');
+
+/**
+ * Convert snake cased json object to protobuf message (factory)
+ *
+ * @param MessageClass
+ *
+ * @returns {jsonToProtobuf}
+ */
+function jsonToProtobufFactory(MessageClass) {
+  /**
+   * Convert snake cased json object to protobuf message
+   *
+   * @typedef jsonToProtobuf
+   *
+   * @param {Object} object
+   *
+   * @returns {jspb.Message}
+   */
+  function jsonToProtobuf(object) {
```

Comment:
I would remove Factory and accept message class as the second param.

Reply:
I would leave it as is, as it is easier to pre-configure and just use conversion method inside of the interceptor and server handler wrapper.